kong/clustering/init.lua

```diff
@@ -285,7 +285,11 @@ function _M:init_worker()
 
       self:request_version_negotiation()
 
-      local config_proto, msg = version_negotiation.get_negotiated_service("config")
+      local config_proto, msg = self.conf.cluster_protocol, "set by configuration"
```

Comment:
Although there is no problem now, I think it is still necessary to check if the protocol specified by `self.conf.cluster_protocol` is accepted by CP.

I think we need to keep all the protocols that are accepted by the CP to get this done.